tests/test_flows.py

```diff
@@ -383,17 +383,33 @@ def test_channel_selection(self):
         self.check_log(self.repeater1, "agent_wlan2", "CHANNEL_PREFERENCE_QUERY_MESSAGE")
 
         self.debug("Send channel selection request")
-        self.gateway_ucc.dev_send_1905(self.mac_repeater1, 0x8006)
+
+        payload_wlan0="0x14 {0x51 {0x0C {0x01 0x02 0x03 0x04 0x05 0x07 0x08 0x09 0x0A 0x0B 0x0C 0x0D} 0x00}} {0x52 {0x00 0x00}} {0x53 {0x08 {0x01 0x02 0x03 0x04 0x05 0x07 0x08 0x09} 0x00}} {0x54 {0x08 {0x05 0x07 0x08 0x09 0x0A 0x0B 0x0C 0x0D} 0x00}} {0x73 {0x00 0x00}} {0x74 {0x00 0x00}} {0x75 {0x00 0x00}} {0x76 {0x00 0x00}} {0x77 {0x00 0x00}} {0x78 {0x00 0x00}} {0x79 {0x00 0x00}} {0x7A {0x00 0x00}} {0x7B {0x00 0x00}} {0x7C {0x00 0x00}} {0x7D {0x00 0x00}} {0x7E {0x00 0x00}} {0x7F {0x00 0x00}} {0x80 {0x00 0x00}} {0x81 {0x00 0x00}} {0x82 {0x00 0x00}}"
```

Comment:
Please add comment to the test explaining what are the steps taken.
Also for such payloads, please add a comment explaining it.

I guess both payloads are channel selection requests, one to change to channel 6 and one to channel 36?

Reply:
Yes. payload for wlan0 - request for change on channel 6, payload for wlan2 - request for change on channel 36